src/Services/Tesseract.php

```diff
@@ -6,22 +6,44 @@ class Tesseract extends OcrAbstract
 {
     protected $command;
 
+    protected function getExecutable()
+    {
+        return config('lara_ocr.engines.tesseract.executable', 'tesseract');
+    }
+
+    public function supportedLanguages()
+    {
+        $shell = new Shell();
+
+        $command = trim($this->getExecutable().' --list-langs');
+
+        $langArray = explode("\n", $shell->execute($command));
+        array_shift($langArray);  // remove heading "List of available languages" from command result
+        
+        return $langArray;
+    }
+
+    public function languageSupported($lang)
```

Comment:
I preffer to use `is` or `has` prefix for methods that return boolean value. So, my recommended method name for this one will be `isLanguageSupported($lang)`